content/docs/user-guide/interactivity/user-interface/canvases/placing-canvases-3d.md

```diff
@@ -1,28 +1,40 @@
 ---
 linkTitle: Placing UI Canvases in the 3D World
-description: ' Place a UI canvas on an object in the 3D world in Open 3D Engine. '
 title: Placing UI Canvases in the 3D World
+description: Place a UI canvas on an object in the 3D world in Open 3D Engine (O3DE).
 weight: 500
 ---
 
-You can place a UI canvas directly on an object in the 3D world, as opposed to showing it in screen space. To do this, you render a UI canvas to a texture, and then use that texture in a material on a 3D mesh.
+You can place a UI canvas directly on an object in the 3D world, as opposed to showing it in screen space. To do this, you render a UI canvas to a texture, and then use that texture in a material of a mesh or actor.
 
-You can use any material on a component entity to display a texture rendered by a UI canvas. If players are to interact with the UI canvas in the 3D world—by clicking with the mouse, for example—an extra **UI Canvas on Mesh** component is required to be added to the entity.
+You can use any material on an entity to display a texture rendered by a UI canvas. To interact with a canvas on a mesh or actor, the entity requires a **UI Canvas on Mesh** component.
 
-Follow all the steps in the following procedure if you need to create a canvas that players can interact with. If the canvas is not to be interactive, then you only need steps 1 through 5.
+## Placing a canvas in the world
 
-**To place a UI canvas on an object in the 3D world**
+If the canvas is not interactable, step 8 of the following procedure is not required.
```

Comment:
I think it's more relevant and sufficient to call this out only in step 8 (as you do already). There's no apparent reason why a user should be aware of this ahead of time.
```suggestion
```